app/plugins/Wallet/js/lifecycle.js

```diff
@@ -197,28 +197,9 @@ addResultListener('update-history', function(result) {
 function start() {
 	// DEVTOOL: uncomment to bring up devtools on plugin view
 	// IPC.sendToHost('devtools');
-	
-	// Need to check if wallet's unencrypted
-	IPC.sendToHost('api-call', '/wallet', 'on-opened');
-}
-
-// First status call to diagnose the state of the wallet
-addResultListener('on-opened', function(result) {
-	wallet = result;
 
-	// Show correct lock status. TODO: If the wallet is encrypted, prompt with
-	// a pw.
-	if (!wallet.encrypted) {
-		setUnencrypted();
-   	} else if (!wallet.unlocked) {
-		setLocked();
-	} else if (wallet.unlocked) {
-		setUnlocked();
-	}
-
-	// Start updating
 	update();
-});
+}
```

Comment:
suspicious...

But looks like it'll work. I'm just remembering we had problems the last time this diff was introduced. But here it's closing a standard function instead of a closure.